src/sdk/namespace_fs.js

```diff
@@ -79,6 +79,46 @@ function sort_entries_by_name(a, b) {
     return 0;
 }
 
+function _get_mtime_from_filename(filename) {
+    if (!filename.includes('_mtime-')) {
+        // Latest file wont have time suffix which will push the latest
+        // object last in the list. So to keep the order maintained,
+        // returning the latest time. Multiplying with 1e6 to provide
+        // nano second precision
+        return Date.now() * 1e6;
```

Comment:
not sure, but you might need to return it as a `size_utils.BigInterger` for consistency 

```suggestion
        return size_utils.BigInterger(Date.now() * 1e6);
```

Reply:
sure